runner/sidecar/sources.go

```diff
@@ -54,6 +54,16 @@ func connectSources(ctx context.Context, process func(context.Context, []byte) e
 		Help:      "Total number of bytes processed, see https://github.com/argoproj-labs/argo-dataflow/blob/main/docs/METRICS.md#sources_retries",
 	}, []string{"sourceName", "replica"})
 
+	initMetrics := func(sourceName string, replica int) {
```

Comment:
do we want to set these to zero? I think it might cause the graphs issues, better undefined? I don't know if this is correct or not actually.

Reply:
yeah, undefined is more accurate. Let me fix the test case then.